pipeline/sfm.cc

```diff
@@ -50,17 +50,23 @@ Reconstruction triangulate_points(Reconstruction reconstruction,
 std::map<size_t, Reconstruction> incremental_mapping(
     const py::object database_path_,
     const py::object image_path_,
+    const py::object input_path_,
```

Comment:
Maybe we can move this as last parameter with default empty from python string such that we don't break compatibility.

Reply:
Yes 👍